kolibri/plugins/coach/assets/src/views/plan/CoachExamsPage/index.vue

```diff
@@ -7,153 +7,157 @@
   >
 
     <KPageContainer>
-      <PlanHeader />
-
-      <div class="filter-and-button">
-        <p v-if="filteredExams.length && filteredExams.length > 0">
-          {{ $tr('totalQuizSize', { size: calcTotalSizeOfVisibleQuizzes }) }}
-        </p>
-        <KSelect
-          v-model="statusSelected"
-          :label="coachString('filterQuizStatus')"
-          :options="statusOptions"
-          :inline="true"
-        />
-        <KButtonGroup v-if="practiceQuizzesExist">
-          <KButton
-            primary
-            hasDropdown
-            appearance="raised-button"
-            :text="coachString('newQuizAction')"
-          >
-            <template #menu>
-              <KDropdownMenu
-                :options="dropdownOptions"
-                class="options-btn"
-                @select="handleSelect"
-              />
-            </template>
-          </KButton>
-        </KButtonGroup>
-        <div
-          v-else
-          class="button"
-        >
-          <KRouterLink
-            :primary="true"
-            appearance="raised-button"
-            :to="newExamRoute"
-            :text="coachString('newQuizAction')"
+      <PlanHeader :activeTabId="PlanTabs.QUIZZES" />
+      <KTabsPanel
+        :tabsId="PLAN_TABS_ID"
+        :activeTabId="PlanTabs.QUIZZES"
+      >
+        <div class="filter-and-button">
```

Comment:
If that'd make review easier, I'm fairly confident I didn't make any updates to the code that is now newly wrapped within `KTabsPanel`, and these large diffs are caused by deeper nesting that resulted from that. This applies to all other files containing `KTabsPanel`.